### ISSUE TITLE
signal: text overflow does not truncate with ellipsis 

### ISSUE THREAD

**Issue by mkchang** [eBay/skin#2380]
### I verified there's no existing issue for this bug.

- [X] There are no existing issues

### Current behavior

Longer text in a signal does not truncate into ellipsis
![image](https://github.com/user-attachments/assets/92ea19be-21aa-47a4-9875-f6f4581bccbf)

### Expected behavior

Expected truncation per DS specs:
![image](https://github.com/user-attachments/assets/5f356e09-f4d1-4900-8a37-fc5b80e02af8)

### Skin version

18.0.4

### Device/OS/Browser info

MacOS + Chrome 126.0.6478.127

### Steps to reproduce

```bash
1. Using ebayui-core/ebay-signal, inserted text that was longer than the allotted space for the signal.
2. See text wrap instead of truncate.
```

**Comment by ArtBlue** [comment]
The Signal component in Figma is `v1.2`. What's implemented in Skin is `1.0`. It means we have not implemented this yet. We've actually not had discussions with DS about it either, and, at first glance, this doesn't appear to be an ideal solve. We have a few questions that we will need to have alignment over.

**Comment by ArtBlue** [comment]
After discussing this in WF, we're closing this issue. If DS wants to later implement this change, we can have those discussions then.
